### ISSUE TITLE
In need of a bash wizard

### ISSUE THREAD

**Issue by 1egoman** [1egoman/biome#20]
Are you good at writing bash? I could use some help refactoring Biome. A few things in particular that need updating:
- Change local variables from uppercase to lowercase (and use the `local` keyword).
- I'm pretty sure the convention is to exit with a different error code for each type of error. Right now, if there's an error, the error code is 1.
- There's a lot of code like `echo hello world` - this should really be `echo "hello world"`.
- Anything else that I missed?

Thanks for the help!

**Comment by rozuur** [comment]
Can I work on this?

**Comment by 1egoman** [comment]
Go for it!

**Comment by 1egoman** [comment]
Yep, I've just recently learned about shellcheck, and most likely there's a lot of small changes that could be made to make this tool comply with typical bash conventions.

**Comment by rozuur** [comment]
I was planning to use shellcheck for refactoring. Will send a pull request by EOW.

**Comment by rozuur** [comment]
@1egoman I am new to circle ci and doesn't know how to trigger the build. Can you check if shellcheck is invoked with the current pull request?

**Comment by Furinkazan33** [comment]
Can i contribute ?

**Comment by 1egoman** [comment]
@Furinkazan33 Go for it!

**Comment by Furinkazan33** [comment]
@1egoman I need the permission to PR.

**Comment by 1egoman** [comment]
@Furinkazan33 Are you saying you need permission to create a pull request? You should be able to fork this project, perform any changes, and issue a pull request without special permission. Maybe I misunderstand what you're asking?
